devices/HumanDataCollector/HumanDataCollector.cpp

```diff
@@ -286,13 +456,79 @@ void HumanDataCollector::run()
 
         }
 
+        // Initialize  matio logging
+        if (pImpl->matioLogger) {
+
+            // Get the start time
+            pImpl->startTime = std::chrono::system_clock::now();
+
+            // TODO: Handle multiple files
+            std::string matLogFileFullPathName = pImpl->matLogDirectory + pImpl->matLogFileName;
+            pImpl->matFilePtr = Mat_CreateVer(matLogFileFullPathName.c_str(), nullptr, MAT_FT_MAT73);
```

Comment:
what happen here if the file already exists? is it overwritten or is it crashing (I see the multiple file handling is still a TODO, I was just curios about the current behaviour)

Reply:
In the current implementation, if the file already exists, it  will be rewritten without causing any errors. One way to proceed for multiple file handling is to append the files with data and time.